src/middleware.js

```diff
@@ -42,7 +55,12 @@ function addRequestSizeMiddleware(app) {
   app.use(requestSizeMiddleware());
 }
 
+function addLocalhostMiddleware(app) {
+  app.use(blockLocalhostMiddleware());
```

Comment:
This will block incoming right? Does it block outgoing?

Reply:
This blocks incoming requests. Incoming requests originate from 10.0.0.0/24. Request generated within from transformer eg: user transform code will have ip 127.0.0.1

Reply:
So are you trying to prevent, user transformation calling transformer?

Reply:
yeah, directed towards ut

Reply:
What happens when we run the transformer locally and call from the postman

Reply:
requests sent on port forwarding pods are also blocked by this